fili-core/src/main/java/com/yahoo/bard/webservice/sql/helper/DatabaseHelper.java

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.helper;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.JDBCType;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * Various functions to help with interacting with a database.
+ */
+public class DatabaseHelper {
+    private static final int DATA_TYPE = 5;
+    private static final int COLUMN_NAME = 4;
+
+    /**
+     * Private constructor - all methods static.
+     */
+    private DatabaseHelper() {
+    }
+
+    /**
+     * Finds the name of the Timestamp column in a sql database.
+     *
+     * @param connection  The connection to the database.
+     * @param schema  The name of the schema (i.e. "PUBLIC").
+     * @param table  The name of the table (i.e. "table").
+     *
+     * @return the name of the timestamp column.
+     *
+     * @throws SQLException if failed while reading database.
+     * @throws IllegalStateException if no {@link JDBCType#TIMESTAMP} column could be found.
+     */
+    public static String getTimestampColumn(Connection connection, String schema, String table)
```

Comment:
That's because specifying the name of the timestamp column doesn't belong in `application.config`. That file is for system wide properties and as you point out, the name of the timestamp column is specific to each table.

However, we already have a mechanism for configuring things that are specific to tables. It's called the table configuration (i.e. the `TableLoader`)! 

I propose we extend `Schema` to include the concept of a timestamp column. The current `Schema` would become the `DruidSchema` or something similar, since Druid tables don't need the timestamp column specified.

We could also include a default value, so that customers don't have to manually specify the timestamp column for every SQL table if they follow some convention (i.e. the table's name is timestamp). @michael-mclawhorn @cdeszaq thoughts?